### DISCUSSION TITLE
CHANGELOG.md out-of-date?

### DISCUSSION THREAD

**Discussion by manchicken** [XAMPPRocky/octocrab#298] [General]
It looks like the `CHANGELOG.md` is out-of-date. Is this just me? It looks like it hasn't been updated for a while.

**Comment by XAMPPRocky** [comment]
Thank you for your question! Yes, I just don't have the time to do it manually anymore, see https://github.com/XAMPPRocky/octocrab/issues/293 for details on automating it.

**Reply by manchicken** [reply]
Thanks!

**Comment by manchicken** [comment]
It looks like you automated this. Sorry, I should have closed this earlier.
